actionpack/lib/action_controller/renderer.rb

```diff
@@ -71,6 +71,18 @@ def initialize(controller, env, defaults)
     end
 
     # Render templates with any options from ActionController::Base#render_to_string.
+    #
+    # The primary options are:
+    # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
```

Comment:
Would partial rendering be considered a main option for controller level rendering? I usually do this in views only.

Reply:
Ive rendered partials outside of a view. It is a thing people do.